components/side-by-side/side-by-side.scss

```diff
@@ -0,0 +1,155 @@
+// Side by Side Component
+// Leverages design patterns from ilr-base and global styles
+
+@import '../../global/global';
+
+.cu-side-by-side {
+  display: grid;
+  gap: var(--cu-vr2);
+  align-items: center;
+  margin-bottom: var(--cu-vr2);
+
+  // Mobile-first: always stack vertically on small screens
+  grid-template-columns: 1fr;
+
+  .cu-side-by-side__media {
+    order: -1; // Image always appears first on mobile
+  }
+
+  .cu-side-by-side__content {
+    order: 1;
+  }
+
+  // Desktop: side by side layout
+  @include breakpoint(sm) {
+    grid-template-columns: 1fr 1fr;
+    gap: var(--cu-vr3);
+
+    // Reset order for desktop - variants will handle positioning
+    .cu-side-by-side__media {
+      order: initial;
+    }
+
+    .cu-side-by-side__content {
+      order: initial;
+    }
+  }
+}
+
+// Content area styling
+.cu-side-by-side__content {
+  display: flex;
+  flex-direction: column;
+  gap: var(--cu-vr1);
+}
+
+.cu-side-by-side__eyebrow {
+  color: var(--cu-colorscheme-accent);
+  font-size: 0.875rem;
+  font-weight: 600;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  margin: 0;
+}
+
+.cu-side-by-side__heading {
+  margin: 0;
+  color: var(--cu-text-color);
+
+  // Use existing heading styles from the design system
+  font-size: var(--cu-font-size-h2);
+  font-weight: var(--cu-font-weight-bold);
+  line-height: var(--cu-line-height-heading);
+}
+
+.cu-side-by-side__body {
+  margin: 0;
+  color: var(--cu-text-color);
+
+  p {
+    margin-bottom: var(--cu-vr1);
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+}
+
+// Button styling
```

Comment:
Button styling can go away once you use the button component. The hover states might be a nice addition to the button component, but we should do that separately (and discuss with Dave and Built, our design partners). 

Notice that there is a colorscheme toggle in the bottom right of Union. It's a handy reference to confirm that styles are working across colorschemes. In this case, the button styles break the colorscheme, for example.